src/Nethermind/Nethermind.Synchronization/Reporting/SyncReport.cs

```diff
@@ -91,6 +91,7 @@ private void TimerOnElapsed(object? sender, EventArgs e)
             if (_reportId % SyncFullPeersReportFrequency == 0)
             {
                 _syncPeersReport.WriteFullReport();
+                Nethermind.TxPool.TxPool.WriteTxnPoolReport(_logger);
```

Comment:
Not sure if this should be invoked from SyncReport.

Reply:
Moved to txPool